presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -318,12 +327,7 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
         builder.add(
                 new IterativeOptimizer(
                         stats,
-                        ImmutableList.of(new com.facebook.presto.sql.planner.optimizations.PickLayout(metadata)),
-                        new PickTableLayout(metadata).rules()));
-
-        builder.add(
-                new IterativeOptimizer(
-                        stats,
+                        ImmutableList.of(new PickLayout(metadata)),
```

Comment:
Now it looks like `PickLayout` became an alternative to `RemoveEmptyDelete` rule. Weird.